archinstall/default_profiles/desktops/cosmic.py

```diff
@@ -13,9 +13,10 @@ def __init__(self) -> None:
 	def packages(self) -> list[str]:
 		return [
 			"cosmic",
+			"xdg-user-dirs"
 		]
 
 	@property
 	@override
 	def default_greeter_type(self) -> GreeterType:
-		return GreeterType.CosmicSession
+		return GreeterType.CosmicGreeter
```

Comment:
If you can revert this change, we can merge in the `xdg-users-dirs` change.
And we can fix the greeter separately.